### ISSUE TITLE
auto fill only .meta

### ISSUE THREAD

**Issue by gutierri** [passff/passff#346]
I believe we could have an autofill option only for .meta files. Today if I enable the autofil option it obligatorily asks for the password every time of the match, which could be only in .meta first.

ref (8f8dbc6)

**Comment by tuxor1337** [comment]
Would you please elaborate on this, e.g. describe step by step the user interaction? Do you mean that we should check if an item has a corresponding `.meta` file before autofilling? Why would anybody want this?

**Comment by gutierri** [comment]
Why auto fill works really well today if you leave gpg-agent with a longer time to expire, otherwise leaving this option enabled is a problem because every match it opens the authentication popup. Another thing is that non-sensitive information can already fill in the fields, for example, email field or service url.

The idea would be to have only one option that prioritizes the autofill in .meta files, if it has not ignores. Just as it is today, but without ever asking for authentication.


02.07.2018, 10:49, "Thomas Vogt" <notifications@github.com>:

**Comment by tuxor1337** [comment]
I understand that, without longer expiration time in the gpg-agent settings, you are asked for authentication quite frequently. But then I don't understand how .meta files would help there. They would have to be decrypted just as well, right?

**Comment by gutierri** [comment]
No, because they do not necessarily need to be encrypted, as it would not do sense to have 2 files encrypted for the same purpose.  From what I understood¹, are files to organize other information that does not need to be in.gpg which would be the encrypted file.In fact, the passff currently encodes the.meta, but it triggers an alert saying that it is not encrypted, would that be a bug?[1] https: //www.passwordstore.org#organization02.07.2018, 18:33, "Thomas Vogt" <notifications@github.com>:I understand that, without longer expiration time in the gpg-agent settings, you are asked for authentication quite frequently. But then I don't understand how .meta files would help there. They would have to be decrypted just as well, right?

**Comment by tuxor1337** [comment]
I can't see any note in the pass documentation about unencrypted entries and it doesn't sound to me like they suggest to have unencrypted .meta files. How do you create unencrypted entries with pass?

**Comment by gutierri** [comment]
I have currently done this with a script, but only non-sensitive data.

But on second thought, it may not make much sense. I believe that, perhaps, would break the sense of the pass.


03.07.2018, 00:11, "Thomas Vogt" <notifications@github.com>:

**Comment by tuxor1337** [comment]
Okay, if it's your private script, I don't think it's desirable to support it in PassFF. I suggest to close this issue.
